jaeger-core/src/main/java/io/jaegertracing/internal/JaegerSpanContext.java

```diff
@@ -107,44 +114,22 @@ public String toString() {
     return contextAsString();
   }
 
-  public static JaegerSpanContext contextFromString(String value)
```

Comment:
why is this still here? Didn't you merge it as a separate PR already? Maybe rebase from master?

Reply:
No just haven't gotten to fixing it yet. Sorry.